### PR TITLE
desktop: revisit how we handle checksums

### PR DESCRIPTION
**Pull request by maxcleme**
### Proposed changes

+ Prepare upcoming changes of how we are handling Docker Desktop checksums (from hardcoded value to hosted files). 
+ Opportunistic refactor of the download sections in order to get identical style everywhere.
+ All checksums files have been generated.

### CODE REVIEW

File: desktop/release-notes.md [hunk 1]
```diff
@@ -1619,10 +1104,7 @@ Alternatively, you can edit the Docker Desktop settings file located at `%APPDAT
 
 {% include release-date.html date="2022-02-10" %}
 
-> Download Docker Desktop
->
-> [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/74594/Docker.dmg) |
-> [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/74594/Docker.dmg)
+{% include desktop-install.md win=true mac=true build_path="/74594/" %}
```
Comment (maxcleme) [inline desktop/release-notes.md:1107]: Windows is currently missing for `4.5.0` release, is it intentional?

**Comment by aevesdocker** [comment]
I feel much calmer looking at this page now. Two picky points:
1. Can we have the checksum files open in a new tab? 
2. I wonder if we need to have a way to visually distinguish between the latest release and the rest? They all look the same now and it all starts to blur. Maybe have the background text be green? (add {: .tip} to the bottom of the callout)

**Comment by aevesdocker** [comment]
Awesome, merci @maxcleme. Ready for this to be merged now?

**Review by aevesdocker — APPROVED** [review]

**Comment by maxcleme** [comment]
@aevesdocker Done!

**Pull request merged**